src/components/Questionnaire/QuestionTypes/ChoiceQuestion.tsx

```diff
@@ -44,7 +48,7 @@ export const ChoiceQuestion = memo(function ChoiceQuestion({
       : "radio";
   const currentValue = questionnaireResponse.values[index]?.value?.toString();
   const currentCoding = questionnaireResponse.values[index]?.coding;
-
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
it'd have been better to make the state's name more verbose since this is specific for valueset alone.
```suggestion
  const [isValuesetSelectOpen, setIsValuesetSelectOpen] = useState(false);
```